spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -75,6 +77,8 @@ interface ChannelEnd {
   counterpartyChannelIdentifier: Identifier
   connectionHops: [Identifier]
   version: string
+  sequence: uint64
+  flushState: FlushStatus
```

Comment:
Is this intended to be `flushState` as the field name, or should it also be `flushStatus`?